opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -210,48 +195,20 @@ def __init__(
         event_name: str | None = None,
     ): ...
 
-    @overload
-    @deprecated(
```

Comment:
This is changing the actual class name so I think is a good opportunity to get rid of previously deprecated parameters